readthedocs/restapi/templates/restapi/log.txt

```diff
@@ -1,10 +1,9 @@
-RTD build information start
+RTD build information
 Build: {{ build.pk }}
 Project: {{ build.project.slug }}
 Version: {{ build.version.verbose_name }} ({{ build.commit }})
 Date: {{ build.date }}
 Success: {{ build.success }}
```

Comment:
What happen if you access this endpoint while the build is still buidling? Should this be allowed? Should we show the status here also?

Reply:
and do not show the `Success` at while builing?

Reply:
If you enter while building you get the log whit the current commands that were executed. I don't know if we want to consider this case so much, since the url to view the raw log is displayed only when the build is finished. But if someone (using the api) wants to access to this endpoint, probably should check the current status with the other endpoint, what you think? I like the idea of not showing the success text, but I feel like we should display something else then.

Reply:
This is an endpoint, so this will more useful to hit from a script instead from a web browser.

So, I suggest to show the status of build but also the success when it's finished.